### ISSUE TITLE
Remove auto-translation path parameter

### ISSUE THREAD

**Issue by KaKi87** [tom-james-watson/old-reddit-redirect#127]
Hi,

Reddit started to annoyingly auto-translate posts for SEO purposes, like the following :

![](https://github.com/tom-james-watson/old-reddit-redirect/assets/21284089/76a74468-481f-457c-bf43-ea7aaec9bca0)

With the following link : https://www.reddit.com/r/selfhosted/comments/175mw87/netcup_vaut_le_coup_en_2023/fr/

Redirected by the extension to the following : https://old.reddit.com/r/selfhosted/comments/175mw87/netcup_vaut_le_coup_en_2023/fr/

Which returns 404, therefore I'd like to suggest automatically removing the language path parameter (in this case : `/fr/`) at the end of the URL.

Thanks

**Comment by tom-james-watson** [comment]
Pull requests welcome here! 🫶

**Comment by KaKi87** [comment]
Actually, things changed in the meantime : translation language is no longer passed as a path parameter (e.g. `/fr`) but as a query parameter (e.g. `/?tl=fr`), which doesn't break our dear Old Reddit.

We avoided work that would have been rendered useless in a month :+1:

**Comment by anotherublockuser** [comment]
@tom-james-watson Hello, i recently stumbled across this and found this issue.

In fact the query string does break the intended functionality.

Simple test with the url from above;: https://www.reddit.com/r/selfhosted/comments/175mw87/netcup_worth_it_in_2023/?tl=fr

IMO the issue is valid and auto-translation query parameter should be stripped from the url so the old design is always used. Please consider reopening this here or in a new ticket.

**Comment by KaKi87** [comment]
Indeed, that's exactly the cause of [me not seeing the old design since the recent change](https://github.com/tom-james-watson/old-reddit-redirect/issues/205#issuecomment-5388021915). I confirm that when the issue occurs, on Vivaldi (Chrome) & Zen (Firefox) as I mentioned, removing the `tl` query parameter fixes it.

**Comment by tom-james-watson** [comment]
This will go out in 3.0.3. It's currently awaiting review.
